src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -449,6 +456,20 @@ public virtual VerifyResult VerifyStateIndependent(ProtocolSettings settings)
             return VerifyResult.Succeed;
         }
 
+        private bool VerifyConflicts(IEnumerable<Transaction> mempool)
+        {
+            return !GetAttributes<ConflictAttribute>()
+                .Select(p => p.Hash)
+                .Intersect(mempool.Select(p => p.Hash))
+                .Any() &&
+                !mempool
+                .Select(p => p
+                    .GetAttributes<ConflictAttribute>()
+                    .Select(p => p.Hash))
+                .Aggregate(Enumerable.Empty<UInt256>(), (conflicts, p) => conflicts.Union(p))
+                .Contains(Hash);
+        }
```

Comment:
This method doesn't take into account network fees and senders of conflicting transactions, thus, its behaviour is not correct. Can we remove it? We have `private bool CheckConflicts(Transaction tx)` inside the mempool, isn't it enough?